app/lib/features/chat/providers/chat_providers.dart

```diff
@@ -280,9 +280,9 @@ final allowSendInputProvider = StateProvider.family.autoDispose<bool, String>(
 );
 
 // whether user has enough permissions to send message in room
-final canSendMessageProvider = FutureProvider.family<bool, String>(
+final canSendMessageProvider = FutureProvider.family<bool?, String>(
   (ref, roomId) async {
     final membership = ref.watch(roomMembershipProvider(roomId));
-    return membership.valueOrNull?.canString('CanSendChatMessages') == true;
```

Comment:
This was the one source of tests failing as it seemed our input states logic is dependent on nullable for loading instance. So I had to revert it... (came from Feedback review change)